Food_Tracker_App/app/src/main/res/layout/recipe_ingredient_content_edit.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:weightSum="4">
+
+
+    <TextView
+        android:id="@+id/Name"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1.4"
+        android:padding = "10dp"
+        android:text="name" />
+
+    <TextView
+        android:id="@+id/Amount"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="0.8"
+        android:padding = "10dp"
+        android:text="amount" />
+
+    <TextView
+        android:id="@+id/Unit"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="0.8"
+        android:padding = "10dp"
+        android:text="unit" />
+
+    <TextView
+        android:id="@+id/Category"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
+        android:padding = "10dp"
+        android:text="category" />
+
+    <Button
+        android:id="@+id/delete_Button"
+        android:layout_width="48dp"
```

Comment:
You have to set the layout width to 0dp for the `layout_weight` attribute to take effect